.gitignore

```diff
@@ -1,11 +1,14 @@
 *~
 *.pyc
 *.DS_Store
-/docs/*/build/*
+/docs/*build/*
 /src/__pycache__/*
 /src/static/*
 /src/.coverage
+/swagger/*index.html
```

Comment:
Is this necessary?

Reply:
gitignore didn't match swagger/index.html and swagger/v2/index.html with swagger/*index.html, so had to add two different lines for those.